### ISSUE TITLE
A11y: length range slider lacks visible focus indication and accessible name

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1458]
### Steps To Reproduce

For focus indication:

1. Navigating with the keyboard, go to "View > Password Generator", open "Options"
2. Set focus (using `Tab`/`Shift+Tab`) to the Length range slider

---

For accessible name:

1. Run a screenreader (e.g. NVDA on Windows)
2. go to "View > Password Generator", open "Options"
3. Set focus (using `Tab`/`Shift+Tab`) to the Length range slider

### Expected Result

For focus indication:

- expect to see a visible focus ring or indication on the range slider when it has focus

---

For accessible name:

- expect the slider to be announced with a specific label/name

### Actual Result

For focus indication:

- the range slider lacks any visible focus indication

---

For accessible name:

- the slider is simply announced as an unlabelled/generic "slider"

### Screenshots or Videos

For focus indication: animated gif showing part of the focus cycle in the password generation options. Note how when the range slider receives focus, there is no visible focus indication on the input itself.
![bitwarden-range-slider-focus](https://user-images.githubusercontent.com/895831/160306265-00dadcd7-c7b2-4429-9252-8ca5f44d7573.gif)

---

For accessible name: recording using NVDA on Windows, with Speech Viewer enabled. Note how the number/spinbox is announced correctly as "Length spin button ...", while the range slider itself is simply announced as an unlabelled "slider".

https://user-images.githubusercontent.com/895831/160306652-cd271edd-2bd5-4164-9132-87b2d9842a12.mp4

### Additional Context

I would actually suggest that the range slider should not be keyboard-focusable at all, as it is redundant for a keyboard user (since they can operate the preceding spin box/number input). This neatly sidesteps the issue of lack of visible focus indication.

For screen reader users though, I'd still suggest adding an explicit accessible name to the non-focusable range slider (thinking for instance of situations where a user may be partially sighted, use a mouse, but also a screen reader ... they'll want to hear the accessible name of the control once they're operating it/it's under their mouse). A simple `aria-label`, replicating the "Length" label, should be sufficient.

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.32.1 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64
